content/sse/index.textile

```diff
@@ -0,0 +1,119 @@
+---
+title: SSE and Raw HTTP Streaming API
+section: streaming-api
+index: 0
+api_separator:
+jump_to:
+  Help with:
+    - Authentication#authentication
+    - Connection state recovery#connection-state-recovery
+  API Routes:
+    - Server-sent events#sse
+    - Plain event stream#event-stream
+---
+
+The Ably SSE and raw HTTP streaming API provides a way to get a realtime stream of events from Ably in circumstances where using a full Ably Realtime client library, or even an MQTT library, is impractical.
+
+HTTP streaming allows for a request from a client to be held by a server, allowing it to push data to the client without further requests. This, much like WebSockets, help avoid the overhead involved in normal HTTP requests. "Server-sent events":https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events (SSE) is another method of achieving this for web pages, usually done through the use of "EventSource":https://developer.mozilla.org/en-US/docs/Web/API/EventSource.
+
+This feature is currently in beta. If you're interested in using this feature, or have suggestions on how we can improve it, please "get in touch":https://www.ably.io/contact.
+
+It is subscribe-only: you can not interact with the channel, including to publish, enter presence, query the presence set, attach and detach from channels (without closing and re-opening the stream), or anything else.
+
+Where possible, we strongly recommend the use of one of our "Realtime client libraries":https://www.ably.io/download, which provide "more features and higher reliability":https://support.ably.io/solution/articles/3000061848, and the full use of our "normal realtime messaging API":/realtime.
```

Comment:
Should we not say something like.  

"Customers who do not want to use a client library on platforms that support SSE, and only require simple subscribe-only streams, may choose to use SSE because it's an open standard, simple, and requires no SDKs on the client-side.  However, where possible, we strongly recommend..."

Reply:
I think at this point, we should tell people that SSE is incredibly simple, open, and here is a code sample showing how easy it is.  With a link to JSBin to try it out live.